### ISSUE TITLE
Style tag contents appear on main page

### ISSUE THREAD

**Issue by eddiezane** [TryGhost/Ghost#5620]
This might belong in casper, but the following content appears as the picture below in the post preview on the homepage.

```
<style type="text/css">
.content {
  text-align: center;
}
</style>

<h1 class="content">Hello</h1>
```

![screen shot 2015-07-31 at 7 40 45 pm](https://cloud.githubusercontent.com/assets/959573/9019583/2f6e2990-37bc-11e5-823d-8998e06ce4d3.png)

**Comment by acburdine** [comment]
Good catch! I think I found a solution that fixes the issue :smile:

**Comment by ErisDS** [comment]
Hey @eddiezane, the immediate work around for this is to include your `<style>` block later in your post, so that it doesn't get caught by the excerpt. This bug is a duplicate of #3846, so I'm going to close it as such.

@acburdine if you're looking to fix this, can you review the notes on #3846 and mark your PR as closing that issue instead? It can still reference this one as well :)
